recipe/meta.yaml

```
@@ -1,30 +1,34 @@
-{% set version = "2.3.1" %}
+# https://pagure.io/python-daemon/issue/94
+{% set version = "3.0.1" %}
 
 package:
   name: python-daemon
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/python-daemon/python-daemon-{{ version }}.tar.gz
-  sha256: 15c2c5e2cef563e0a5f98d542b77ba59337380b472975d2b2fd6b8c4d5cf46ca
+  sha256: 6c57452372f7eaff40934a1c03ad1826bf5e793558e87fef49131e6464b4dae5
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
   skip: True  # [win or (linux and s390x)]
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
+    # Install won't work with setuptools >69.5.1
+    - setuptools 69.5.1
```

```suggestion
    - setuptools <=69.5.1
```
To make rebuilds easier.